lib/widgets/action_sheet.dart

```diff
@@ -43,18 +44,19 @@ abstract class MessageActionSheetMenuItemButton extends StatelessWidget {
   }) : assert(messageListContext.findAncestorWidgetOfExactType<MessageListPage>() != null);
 
   IconData get icon;
-  String get label;
+  String Function(ZulipLocalizations) get translateLabel;
```

Comment:
I think the name `label` is still good for this.

The background principle is that translation/i18n should be the normal and expected situation, for any string that appears in the UI. So this is just the method that provides the label. It's now a function that takes an argument — so not just a getter returning `String` — but that's just how it has to be in a world where the UI is appropriately internationalized, so it doesn't call for being highlighted in the name.